packages/client/src/websocket.js

```diff
@@ -18,7 +18,7 @@ export const initWebsocket = () => {
   }
 
   // Initialise connection
-  socket = createWebsocket("/socket/client")
+  socket = createWebsocket("/socket/client", false)
```

Comment:
can we update this boolean parameter to an object specifier? Looking at this in PR i'm not sure what purpose the `false` has/does as part of the websocket creation.

Reply:
I will update this at some stage, but CI is passing and I'm 90% sure I'll need another PR after this anyway, so merging for now!